storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -170,7 +240,6 @@ export const RadioGroupWithValidation: Story = {
   render: (args) => (
     <FieldSet
       aria-describedby={`description-e${args.invalid ? ' error-e' : ''}`}
-      aria-required="true"
```

Comment:
Is this intentional? I'm not sure we want this

Reply:
No longer relevant (but doesn't show "outdated" for some reason).

Reply:
This is still relevant? `aria-required` is removed, I don't think it should be.

Reply:
Added `aria-required` to all Field Sets that only have required Fields.